client/src/flowTypes.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+export type Restaurant = {
+  name?: string,
```

Comment:
name is not mandatory? 
address? cuisine? rating? accepts10Bis?

Reply:
Got confused with the `?`, now I got it :)